content/en/docs/tutorials/acme/ingress.md

```diff
@@ -65,13 +65,19 @@ For more information on securing your installation see: https://docs.helm.sh/usi
 Happy Helming!
 ```
 
+Add the latest helm repository for the ingress-nginx
+```bash
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+```
+
 Update the helm repository with the latest charts:
 
 ```bash
 $ helm repo update
 Hang tight while we grab the latest from your chart repositories...
 ...Skip local chart repository
 ...Successfully got an update from the "stable" chart repository
+...Successfully got an update from the "stable" chart repository
```

Comment:
```suggestion
...Successfully got an update from the "ingress-nginx" chart repository
```

might be more logic

Reply:
good catch 🙏  thank you @meyskens, I have just done it.